src/atoms/person.ts

```diff
@@ -1,10 +1,15 @@
 import { atom } from "jotai";
-import { DailyEntry, Person, PersonTotalTime } from "../generated/client";
-import { DailyEntryWithIndexSignature, PersonWithTotalTime } from "../types";
+import type { DailyEntry, Person, PersonTotalTime } from "../generated/client";
+import type { DailyEntryWithIndexSignature, PersonWithTotalTime } from "../types";
+import type { Allocations, Projects, UsersAvatars } from "src/generated/homeLambdasClient";
 
 export const personsAtom = atom<Person[]>([]);
 export const personTotalTimeAtom = atom<PersonTotalTime | undefined>(undefined);
 export const personsWithTotalTimeAtom = atom<PersonWithTotalTime[]>([]);
 export const timebankScreenPersonTotalTimeAtom = atom<PersonTotalTime | undefined>(undefined);
 export const personDailyEntryAtom = atom<DailyEntryWithIndexSignature | undefined>(undefined);
 export const dailyEntriesAtom = atom<DailyEntry[]>([]);
+export const avatarsAtom = atom<UsersAvatars[]>([]);
+export const timeEntriesAtom = atom<number[]>([]);
+export const allocationsAtom = atom<Allocations[]>([]);
+export const projectsAtom = atom<Projects[]>([]);
```

Comment:
I'm wondering if some of these atoms should be in a different file e.g. a sprint-data.ts or something similar.
Perhaps dailyEntries, timeEntries, allocations and projects could go in there? 
What do you think?